CLAUDE.md

```
@@ -0,0 +1,273 @@
+
+# CLAUDE.md
+
+This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
+
+## Directory and File Overview
+
+### Features Directory (`lib/features/`)
+
+- **device_controller/**
+	- `device_controller_page.dart`: UI for MIDI device management and connection
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)